deploy/util/git.py

```diff
@@ -20,7 +21,7 @@ def git_origin(workdir:str = ".") -> str:
     Returns:
         first fetch url of 'origin' remote in `workdir` or emptystring if it is not defined
     """
-    with StreamLogger('stdout'), StreamLogger('stderr'), lcd(workdir), warn_only():
+    with lcd(workdir), warn_only(), StreamLogger('stdout'), StreamLogger('stderr'):
```

Comment:
Since we're reworking the StreamLogger usage in #1159, let's go ahead and pull them out of all the contexts in this new file.

Reply:
Sounds good, done.